docs/dashboards/cloud-posture.asciidoc

```diff
@@ -1,17 +1,40 @@
 [[cloud-posture-dashboard]]
-// Note: This page is intentionally duplicated by docs/cloud-native-security/cloud-nat-sec-posture.asciidoc. When you update this page, update that page to match.
+// Note: This page is intentionally duplicated by docs/cloud-native-security/cloud-nat-sec-posture.asciidoc. When you update this page, update that page to match. And careful witht the anchor links because they should not match.
```

Comment:
OK, see the previous incarnation of this content for any edits, and apply those here too.

Also fixed a li'l typo in the comment:
```suggestion
// Note: This page is intentionally duplicated by docs/cloud-native-security/cloud-nat-sec-posture.asciidoc. When you update this page, update that page to match. And careful with the anchor links because they should not match.
```